mesonbuild/environment.py

```diff
@@ -195,6 +195,7 @@ def get_llvm_tool_names(tool: str) -> T.List[str]:
     # unless it becomes a stable release.
     suffixes = [
         '', # base (no suffix)
+        '-18',  '18',
```

Comment:
This is only adding 18, not 17 now (so commit message needs tweaking).

Reply:
Thanks, our rebase tool automatically fixed the conflict so we didn't notice that.